tutorials/algorithms/03_vqe_simulation_with_noise.ipynb

```diff
@@ -349,57 +351,66 @@
     }
    ],
    "source": [
-    "if counts2 or values2:\n",
-    "    pylab.rcParams['figure.figsize'] = (12, 4)\n",
-    "    pylab.plot(counts2, values2)\n",
-    "    pylab.xlabel('Eval count')\n",
-    "    pylab.ylabel('Energy')\n",
-    "    pylab.title('Convergence with noise, measurement error mitigation enabled')"
+    "if counts or values:\n",
+    "    pylab.rcParams[\"figure.figsize\"] = (12, 4)\n",
+    "    pylab.plot(counts, values)\n",
+    "    pylab.xlabel(\"Eval count\")\n",
+    "    pylab.ylabel(\"Energy\")\n",
+    "    pylab.title(\"Convergence with noise\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Summary\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Lets bring the results together here for a summary. \n",
+    "We produced a reference value using a classical algorithm and then proceeded to run VQE on an Aer simulator. While the simulation is ideal (no noise) there is so called shot-noise due to sampling - increasing the number of shots reduces this as more and more samples are taken, but shots were left at the default of 1024 and we see a small effect in the outcome. \n",
```

Comment:
Well for the Estimator shots was left at the default of None. Does this internally default to 1024 - that is certainly what the defaults when you used shots in the past to run circuits directly. I did not see anything in the Aer Estimator docs as to what to expect - maybe it drops through to that same default.

Reply:
Mmm true, I did not look very deep into the Aer Estimator default, I will test it real quick.